firebase-installations/src/main/java/com/google/firebase/installations/remote/FirebaseInstallationServiceClient.java

```diff
@@ -486,4 +495,19 @@ static long parseTokenExpirationTimestamp(String expiresIn) {
         ? 0L
         : Long.parseLong(expiresIn.substring(0, expiresIn.length() - 1));
   }
+
+  // Read the error message from the response.
+  private String readErrorResponse(HttpURLConnection conn) throws IOException {
+    try (BufferedReader reader =
+        new BufferedReader(new InputStreamReader(conn.getErrorStream(), UTF_8))) {
+      StringBuilder response = new StringBuilder();
+      for (String input = reader.readLine(); input != null; input = reader.readLine()) {
+        response.append(input).append('\n');
+      }
+
+      return String.format(
+          "The FIS server responded with an error. HTTP response: [%d %s %s]",
```

Comment:
**FIS**
"Firebase Installations API" or "Firebase Installations server API" instead of "FIS server".
We don't use FIS in public documentation.
Also it uses the term "Firebase" so developers are more likely to recognize that this is used by their Firebase service.

**Colon**
Please add a colon between ResponseMessage and ResponseBody. I think that is the standard when printing these errors (not 100% sure).

**Suggestion**
```
"Error when communicating with the Firebase Installations server API. HTTP response: [%d %s: %s]"
```